frontend/src/components/about/about.css

```diff
@@ -0,0 +1,45 @@
+.member-image {
+    width: 150px;
+    height: 150px;
+    border-radius: 100%;
+    border-color: #4682b4;
```

Comment:
Replace color with the hex color in figma if using CSS. If done through mui, it can be access by specifying color as "primary" as we already define that in the theme configuration